tiledb/sys/src/stats.rs

```diff
@@ -0,0 +1,25 @@
+extern "C" {
+    #[doc = "Enable internal statistics gathering. \n @return `TILEDB_OK` for success and `TILEDB_ERR` for error."]
```

Comment:
Remove all of the `#[doc = ..]` attributes. We'll need to re-document the Rust API in `tiledb/api` so these don't add anything other than add noise. The other examples including docs just haven't been updated to remove the attributes yet.